geopandas/geoseries.py

```diff
@@ -594,7 +594,30 @@ def _constructor(self):
     def _constructor_expanddim(self):
         from geopandas import GeoDataFrame
 
-        return GeoDataFrame
+        def expanddim(data=None, index=None, crs=None, **kwargs):
+            # if geometry is somehow supplied, ignore it because we don't need it
+            kwargs.pop("geometry", None)
```

Comment:
Are you aware of a test case where this actually was passed?

Reply:
If I was I'm not now... Thinking about it again, I don't think it can be supplied `_constructor_expanddim` isn't used at all by GeoPandas directly, and I don't see how pandas could pass through an arbitrary keyword argument.